front-end-admin/src/components/pages/Home.vue

```diff
@@ -31,15 +30,14 @@
             </table>
           </div>
           <div class="col-md-6">
-            <p><a class="btn btn-default" href="${pageContext.request.contextPath}/registerneed.jsp" role="button">Add Need&raquo;</a></p>
+            <p><router-link to="register-need" class="btn btn-default" role="button">Add Need&raquo;</router-link></p>
```

Comment:
And what does the &raquo do?

Reply:
it's the double >> thing in the button label